tutor/resources/styles/components/task-step/ends.less

```diff
@@ -81,7 +81,8 @@
       content: attr(data-question-number) ")";
       position: absolute;
       z-index: 1;
-      left: -25px;
+      right: 100%;
```

Comment:
Hehe, yeah that was a bit "clever" of me, it set's the right border of the label to the leftmost border of it's container.